Do-It/Model/DoIts/DoIt.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DoIt.swift
+//  Do-It
+//
+//  Created by Michael Pangburn on 11/15/18.
+//  Copyright © 2018 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import Foundation
+
+protocol DoIt {
+    var id : DoItId { get }
```

Comment:
Style nit: colons should immediately follow declarations. Applies to variables and protocol conformances both.